JSDemos/Demos/ButtonGroup/Overview/description.md

```diff
@@ -1,3 +1,17 @@
-ButtonGroup is a UI component that displays a set of toggleable buttons and allows a user to select one or several of them. This demo illustrates how to configure the ButtonGroup to enable single and multiple selection.
+ButtonGroup displays a set of toggleable buttons and allows users to select one or several of them. This demo illustrates how to configure the ButtonGroup: bind ButtonGroup to data, enable or disable multiple selection and customize the ButtonGroup appearance.
+
+## Bind ButtonGroup to Data
+
+Use the [items](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/items/) array to populate ButtonGroup with data. You can specify the following fields in the data objects: [hint](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/items/#hint), [icon](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/items/#icon), [text](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/items/#text), [type](/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/items/#type) and others.
```

Comment:
I've got two notes here:
1. If you say "the following" followed by a list of more than three items, it's better to make a list.
2. If you say "the following", it's not a good practice to make an open-ended list (and others, etc., at the end). You need either to list everything, or just select only the most important entries and pretend as if there's nothing else present.

Make your choice.

Reply:
Got you, I'll make a list.

Reply:
Looks better. Thank you.